### ISSUE TITLE
Doesn't work with a simple npm install

### ISSUE THREAD

**Issue by luandro** [mhnpd/react-loader-spinner#18]
```
journal/node_modules/react-loader-spinner/dist/loader/css/CradleLoader.css:1
(function (exports, require, module, __filename, __dirname) { .swing div {
                                                              ^

SyntaxError: Unexpected token .
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:139:10)
    at Module._compile (module.js:616:28)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.defineProperty.value (journal/node_modules/react-loader-spinner/dist/loader/CradleLoader.js:5:40)
    at Object.<anonymous> (journal/node_modules/react-loader-spinner/dist/loader/CradleLoader.js:13:3)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.defineProperty.value (journal/node_modules/react-loader-spinner/dist/index.js:5:434)
    at Object.<anonymous> (journal/node_modules/react-loader-spinner/dist/index.js:13:3)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (journal/node_modules/react-loader-spinner/index.js:1:80)
```

**Comment by mikeseese** [comment]
I can confirm this is an issue

**Comment by mhnpd** [comment]
@seesemichaelj  @luandro  Can you post you configuration to recreate this issue. ``` .swing ``` is css class and syntax is technically correct.

**Comment by purohit** [comment]
Same issue here -- what configuration can I post?

**Comment by supukarmin** [comment]
I was able to build with version "2.1.0" (7 months ago), all newer versions are effected by this issue. (note: 2.0.1-2.0.6 is newer than 2.1.0)

**Comment by purohit** [comment]
^ @supukarmin same is true for me, I use 2.1.0 and it works, anything above that fails

**Comment by DaniyalBasit** [comment]
same issue ```Unable to resolve "./css/CradleLoader.css" from "node_modules/react-loader-spinner/dist/loader/CradleLoader.js"````

**Comment by mhnpd** [comment]
@DaniyalBasit can you verify that this file exist inside the node module. Its a simple css file and should be there. Need code to re-create it.

**Comment by DaniyalBasit** [comment]
Yes, the css file is there.

**Comment by mhnpd** [comment]
@DaniyalBasit Let me know if you find a way around with your configuration.

**Comment by DaniyalBasit** [comment]
@mhnpd This is a very weird issue. I have tried different versions only version 2.0.0 works, but then that version has an animation issue.

**Comment by hanchiang** [comment]
Hey guys I encountered this issue too, it was because I didn't configure webpack to parse css properly(was using scss).

So I tweaked the css test from `test: /\.scss$/` to `test: /\.s?css$/`, and it worked.

**Comment by troygoode** [comment]
if you're using `next.js` you'll likely run into this due to zeit/next-plugins#267

**Comment by mhnpd** [comment]
From version ``` 3.1.2 ``` it is necessary to import CSS files for loader from node modules.
``` import "react-loader-spinner/dist/loader/css/react-spinner-loader.css" ``` 
In some configuration importing CSS directly,  dist folder is causing this issue. 
This closes #18
